heareval/predictions/task_predictions.py

```diff
@@ -904,6 +973,40 @@ def hparams_to_json(hparams):
     return {k: serialize_value(v) for k, v in hparams.items()}
 
 
+def data_splits_from_folds(folds: List[str]) -> List[Dict[str, List[str]]]:
+    """
+    Create data splits by using Leave One Out Cross Validation strategy.
+
+    Test, Valid and Train Splits will consist of data folds. Successively treat
+    each fold as test split, the next fold as valid split and the remaining
+    fold as the train split.
+    Folds will be sorted before applying the above strategy
+    Total data splits will be equal to n, n being the total number of folds.
+    Each fold will be tested by training on the remaining folds.
```

Comment:
I'd just write an explicit example of what is returned, key and values

Reply:
or say:

"""
With 5-fold, for example, we would have:
test=fold1, val=fold2, train=fold3..5,
test=fold2, val=fold3, train=fold4..5,1,
...